SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -141,37 +129,21 @@ void USpatialDispatcher::ProcessExternalSchemaOp(Worker_Op* Op)
 	Worker_ComponentId ComponentId = GetComponentId(Op);
 	check(ComponentId != SpatialConstants::INVALID_COMPONENT_ID);
 
-	UOpCallbackTemplate** UserCallbackWrapper = UserOpCallbacks.Find(ComponentId);
-	if (UserCallbackWrapper == nullptr)
-	{
-		return;
-	}
-	UOpCallbackTemplate* UserCallback = *UserCallbackWrapper;
-
 	switch (Op->op_type)
 	{
+	case WORKER_OP_TYPE_AUTHORITY_CHANGE:
+		StaticComponentView->OnAuthorityChange(Op->authority_change);
+		// Intentional fall-through
 	case WORKER_OP_TYPE_ADD_COMPONENT:
-		UserCallback->OnAddComponent(Op->add_component);
-		break;
 	case WORKER_OP_TYPE_REMOVE_COMPONENT:
-		UserCallback->OnRemoveComponent(Op->remove_component);
-		break;
 	case WORKER_OP_TYPE_COMPONENT_UPDATE:
-		UserCallback->OnComponentUpdate(Op->component_update);
-		break;
-	case WORKER_OP_TYPE_AUTHORITY_CHANGE:
-		UserCallback->OnAuthorityChange(Op->authority_change);
-		StaticComponentView->OnAuthorityChange(Op->authority_change);
-		break;
 	case WORKER_OP_TYPE_COMMAND_REQUEST:
-		UserCallback->OnCommandRequest(Op->command_request);
-		break;
 	case WORKER_OP_TYPE_COMMAND_RESPONSE:
-		UserCallback->OnCommandResponse(Op->command_response);
+		RunCallbacks(ComponentId, Op);
 		break;
 	default:
-		// This should only happen if the ComponentId is a valid Id, and
-		// a new type of Op as been added to the worker SDK.
+		// This never happen providing the GetComponentId has the same
```

Comment:
grammar

Reply:
If someone is reading this comment they probably hit the error case - what would they need to know to fix their bug.

Reply:
I think reading this comment prompts them to look at GetComponentId, and if the switch cases don't match, then raise an issue with the GDK, and if they do match, then raise an issue with the GDK but more confused? I think this kind of thing will be more useful when the GDK comes to do automated unit testing of things like the dispatcher, rather than directly valuable to a user. Would you suggest any changes from what it is now? (after having fixed the grammar)